change/@fluentui-react-examples-292153ef-66f4-43d3-b6e7-8817fdcf814c.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
I would not expect deleted change files in the PR. Wrong merge?

Reply:
These have been deleted in master, so maybe something strange happened while merging and github is misinterpreting what the baseline commit is?

Reply:
I manually deleted them as this was suggested by `yarn change`. Reverted.

Reply:
It looks like you need to merge with master anyway (there's a merge conflict). Once you do that, the change files will be gone and won't show up in the diff.